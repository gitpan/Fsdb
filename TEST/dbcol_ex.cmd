prog='dbcol'
args='account'
cmp='diff -cb '
in=TEST/dbrow_ex.in
