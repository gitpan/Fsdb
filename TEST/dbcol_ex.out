#fsdb account
johnh
greg
root
# this is a simple database
#   | dbcol account
